### PR TITLE
Tests itemcontroller

### PR DESCRIPTION
**Pull request by Andreassim**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Description?      | Tests for ItemController
| Type?             |  improvement 
| Fixed ticket?     | ~
| How to test?      | ~

### CODE REVIEW

**Comment by MathiasReker** [comment]
LGTM

File: src/main/java/com/app/models/menuactions/DeleteItemMenuAction.java [hunk 1]
```diff
@@ -10,6 +12,6 @@ public DeleteItemMenuAction(String description) {
 
   @Override
   public void run() {
-    new ItemController().deleteItem();
+    new ItemController().deleteItem(new Scanner(System.in));
```
Comment (MathiasReker) [inline src/main/java/com/app/models/menuactions/DeleteItemMenuAction.java]: Store the scanner in a variable to improve the readability.

Comment (Andreassim) [reply src/main/java/com/app/models/menuactions/DeleteItemMenuAction.java]: have added this change

**Comment by MathiasReker** [comment]
Kan vi udkommentere testen, som fejler og merge resten?

**Review by Moshizzl3 — APPROVED** [review]
LGTM!

**Pull request merged**